src/utils/Paginator/paginator.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import styles from './paginator.module.css';
+
+const paginator = (props) => {
+  const {
+    totalUsersCount, currentPage, onPageChanged, pageSize,
+  } = props;
+  const pagesCount = Math.ceil(totalUsersCount / pageSize);
+
+  const pages = [...Array(pagesCount).keys()];
+
+  const pagesMap = pages.map((page) => {
```

Comment:
Consider keeping the map handler in a separate named function.

Reply:
I will follow this recommendation